src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package kitchenpos.application;
+
+import kitchenpos.application.fixture.MenuGroupFixture;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuGroupRepository;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.*;
+import org.junit.jupiter.params.*;
+import org.junit.jupiter.params.provider.*;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    @Mock
+    private MenuGroupRepository repository;
+
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    @DisplayName("메뉴 그룹을 생성할 수 있다.")
+    @ParameterizedTest
+    @ValueSource(strings = {"메인 메뉴"})
+    void createMenuGroup(String name) {
+
+        MenuGroup menuGroup = MenuGroupFixture.setMenuGroup(name);
+
+        given(repository.save(any())).willReturn(menuGroup);
+
+        MenuGroup resultMenuGroup = menuGroupService.create(menuGroup);
+
+        assertAll(
+                () -> assertThat(resultMenuGroup.getId()).isInstanceOf(UUID.class),
+                () -> assertThat(resultMenuGroup.getName()).isEqualTo(name)
+        );
+    }
```

Comment:
테스트 코드에서는 given-when-then 패턴을 사용합니다. 
각 행동이 어디까지 포함이 되는지 주석으로 간단하게 표시해주시면 이해하기 편리합니다 😄 

```suggestion
    void createMenuGroup(String name) {

        //given
        MenuGroup menuGroup = MenuGroupFixture.setMenuGroup(name);
        given(repository.save(any())).willReturn(menuGroup);

        //when
        MenuGroup resultMenuGroup = menuGroupService.create(menuGroup);

        //then
        assertAll(
                () -> assertThat(resultMenuGroup.getId()).isInstanceOf(UUID.class),
                () -> assertThat(resultMenuGroup.getName()).isEqualTo(name)
        );
    }
```